src/lex_utils.cc

```diff
@@ -238,7 +238,7 @@ TEST_SUITE("Offset") {
 
   TEST_CASE("in middle of content") {
     std::string content = "abcdefghijk";
-    for (int i = 0; i < content.size(); ++i) {
+    for (size_t i = 0; i < content.size(); ++i) {
```

Comment:
I guess this may fail to compile on MSVC because `size_t -> int` seems disallowed by some wscript options.

Reply:
Changed to `int`.